change/@microsoft-teams-js-5f6c016e-96a3-464a-95e1-67d1f38531b0.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
this probably isn't a "none" change, since the behavior will be different for some applications before and after, right? Maybe patch instead?

Reply:
Updated